src/Redux/mission/missionSlice.jsx

```diff
@@ -0,0 +1,48 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+import axios from "axios";
+
+export const fetchMissions = createAsyncThunk('mission/fetchMissions', async () => {
+  try {
+    const data = await axios.get("https://api.spacexdata.com/v3/missions");
+    return data.data;
+  } catch (error) {
+    return error.message;
+  }
+})
+
+const MissionSlice = createSlice({
+  name: "missions",
+  initialState: {missions: [], status: "idle"},
+  reducers: {
+    joinMission: (state, { payload }) => {
+
+    }
+  },
+  extraReducers: (Builder) => {
+    Builder
+    .addCase(fetchMissions.pending, (state) => {
+      state.status = "pending";
+    })
+    .addCase(fetchMissions.fulfilled,(state, action) => {
+      const data = [];
+      action.payload.map(x => {
+        let { mission_id, description, mission_name } = x;
+        data.push({ mission_id, description, mission_name });
+        return data;
+      });
```

Comment:
_[OPTIONAL]_ Since the map function is iterating over the missions array, it will be a huge favour to your code reader to represent each object with a more descriptive identifier instead of `x` 

for example `action.payload.map(x =>{ your code.....})` use `action.payload.map(mission =>{ your code.....})`